pkg/fleet/env/env.go

```diff
@@ -129,19 +130,16 @@ func (e *Env) ToEnv() []string {
 		env = append(env, envRegistryAuth+"="+e.RegistryAuthOverride)
 	}
 	if len(e.ApmLibraries) > 0 {
-		b := strings.Builder{}
-		b.WriteString(envApmLibraries)
-		b.WriteString("=")
+		libraries := []string{}
 		for l, v := range e.ApmLibraries {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** ‏Can we be a bit more explicit with the var names?